dpnp/backend_mathematical.pyx

```diff
@@ -205,7 +205,7 @@ cpdef dparray dpnp_subtract(dparray x1, dparray x2):
     return call_fptr_2in_1out(DPNP_FN_SUBTRACT, x1, x2, x1.shape)
 
 
-cpdef dpnp_sum(dparray x1):
+cpdef dparray _dpnp_sum(dparray x1):
```

Comment:
please rename it to `dpnp_sum_no_axis` or better move it into original function `dpnp_sum`

Reply:
it will allow to reuse `return_type ` instead `res_type`